src/Simplex_tree/include/gudhi/Simplex_tree/Simplex_tree_multi.h

```diff
@@ -0,0 +1,229 @@
+/*    This file is part of the Gudhi Library - https://gudhi.inria.fr/ - which is released under MIT.
+ *    See file LICENSE or go to https://gudhi.inria.fr/licensing/ for full license details.
+ *    Author(s):       David Loiseaux, Hannah Schreiber
+ *
+ *    Copyright (C) 2023 Inria
+ *
+ *    Modification(s):
+ *      - YYYY/MM Author: Description of the modification
+ */
+#ifndef SIMPLEX_TREE_MULTI_H_
```

Comment:
`Simplex_tree_multi_utilities.h` seems more precise and less misleading to me than `Simplex_tree_multi.h`. What do you think?

Reply:
I'm not a big fan of the `utilities` name, but the name can be changed indeed. Let's fix the other issues and we'll decide that once the content of this file is fixed.